cxx-qt-gen/src/generator/naming/qobject.rs

```diff
@@ -0,0 +1,68 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+use crate::generator::naming::CombinedIdent;
+use crate::parser::qobject::ParsedQObject;
+use quote::format_ident;
+use syn::Ident;
+
+/// Names for parts of a Q_OBJECT
+pub struct QObjectName {
```

Comment:
If #215 lands first, then the TCxxQtThread name should be added to the naming here.